# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/naga/CHANGELOG.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/naga/CHANGELOG.md b/third_party/rust/naga/CHANGELOG.md
--- a/third_party/rust/naga/CHANGELOG.md
+++ b/third_party/rust/naga/CHANGELOG.md
@@ -1,10 +1,22 @@
 # Change Log
 
+## v0.6 (2021-08-18)
+  - development release for wgpu-0.10
+  - API:
+    - atomic types and functions
+    - storage access is moved from global variables to the storage class and storage texture type
+    - new built-ins: `primitive_index` and `num_workgroups`
+    - support for multi-sampled depth images
+  - WGSL:
+    - `select()` order of true/false is swapped
+  - HLSL backend is vastly improved and now usable
+  - GLSL frontend is heavily reworked
+
 ## v0.5 (2021-06-18)
   - development release for wgpu-0.9
   - API:
     - barriers
     - dynamic indexing of matrices and arrays is only allowed on variables
     - validator now accepts a list of IR capabilities to allow
     - improved documentation
   - Infrastructure: